1.9/usage/storage/external-storage.md

```diff
@@ -4,6 +4,9 @@ menu_order: 1
 feature_maturity: experimental
 ---
 
+**Warning:** Volume size is specified in GB. There is currently an inaccuracy in the DC/OS GUI where the volume size is marked in MiB. When creating a volume, make sure you specify the number of **GB** you need, not MiB.
```

Comment:
nit: let's be consistent, if we're using `MiB` then we should also use `GiB` instead of `GB`